plugins/org.wso2.developerstudio.humantaskeditor/.gitignore

```diff
@@ -0,0 +1,70 @@
+/bin/
+/bin/
+.classpath
+.project
+.settings
+/target/
+
+/bin/
```

Comment:
The bin folder has been added to the .gitignore file multiple times. Better to remove unnecessary lines.